tasks/install-python3.6.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Add Deadsnakes PPA
+  apt_repository:
+    repo: ppa:deadsnakes/ppa
```

Comment:
@morrismukiri please make installing the PPA, and the name of the PPA configurable using vars (or a single var). I think newer OS versions don't need a PPA for Python 3.

Reply:
Thanks for catching that. Resolved